src/components/RenderList.vue

```diff
@@ -42,50 +42,45 @@ export default {
             name:"Lopatka",
             quantity:4.1,
             price:10.45,
-            vat:5,
-            gros:"",
+            vat:"",
+            gross:"",
             net:"",
            },
            {
             id:2,
             name:"Kielbasa",
             quantity:1.6,
             price:20.99,
-            vat:5,
-            gros:"",
+            vat:"",
+            gross:"",
             net:"",
            },
           ]
         }
       ]
     }
   },
-
-
   methods: {
-       calculateValue(quantity,price) {
+      calculateNetValue(quantity,price) {
          return (quantity*price).toFixed(2);
-      },         
+      },
+      calculateGrossValue(quantity,price,vat) {
+        return ((quantity*price)*vat).toFixed(2);
```

Comment:
is the same as `(quantity * price * vat).toFixed(2)`

Reply:
Use spaces

Reply:
✔️